idelib/dataset.py

```diff
@@ -2223,6 +2223,10 @@ def arrayMinMeanMax(self, startTime=None, endTime=None, padding=0,
                 for m in range(3):
                     xform.inplace(out[m, :, :], out=out[m, :, :])
 
+        for i in range(len(out[0])):
+            if out[0, i, 0] > out[2, i, 0]:
+                out[:, i] = np.flipud(out[:, i])
```

Comment:
For the sake of future developers, could you add a brief comment describing what this is doing? Just a note saying it's to handle inverted axes or something.

Also, totally optional: it might be marginally more efficient to use one of the axes in `out.shape` instead of calling `len()`.